src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
The `nth-child` selector you are using will not dynamically change the number of active stars. It is always set to 4. Consider using a class to specify the number of active stars, and then in your HTML, you can add that class to the `stars` element to control the number of active stars. You could have classes like `.stars--1`, `.stars--2`, etc., each one setting the `background-image` for the appropriate number of child elements.